project-charter.md

```diff
@@ -20,25 +20,25 @@ The mission of the Project is to:
 
 # **2. Governance Board**
 
-1.  The Governance Board (the “GB”) shall be responsible for overall oversight of the Project and coordination of the efforts of any Technical Committees.
+1.  The Governance Board (the “GB”) shall be responsible for overall oversight of the Project and coordination of the efforts of any Technical Committees and Workgroups.
 
 2.  At inception of the Project, the GB voting members consists of those individuals identified on the Project website (https://chaoss.community). Following inception of the Project, the GB will implement procedures and methodologies for the selection of GB voting members from the contributing community.
 
-3.  The GB will encourage transparency (e.g. publish public minutes). Technical Committees meetings will be open to the public, and can be conducted electronically, via teleconference, or in person.
+3.  The GB will encourage transparency (e.g., publish public minutes). Committee and Workgroup meetings will be open to the public, and can be conducted electronically, via teleconference, or in person.
```

Comment:
Why removing "technical" from "technical committee"? I don't really mind using one or the other, but I wonder if there is some rationale for the change....

Reply:
"technical" appeared to serve no purpose. We have committees. Adding 'technical' might imply that we also have other types. I favor removing technical because it simplifies the language without loosing clarity.

Reply:
OK, fine with me.